Description: Disable bluetooth applet hack
 The hack prevent the Debian package to build properly.
Author: Nicolas Bourdaud <nicolas.bourdaud@gmail.com>
Origin: vendor
Forwarded: https://github.com/linuxmint/Cinnamon/pull/987
Applied-Upstream: no
Last-Update: 2012-09-28

--- a/src/cinnamon-plugin.c
+++ b/src/cinnamon-plugin.c
@@ -382,17 +382,3 @@
 
   return &info;
 }
-
-#if HAVE_BLUETOOTH
-/* HACK:
-   Add a non-static function that calls into libgnome-bluetooth-applet.so,
-   to avoid the linker being too smart and removing the dependency.
-   This function is never actually called.
-*/
-extern GType bluetooth_applet_get_type(void);
-void _cinnamon_link_to_bluetooth(void);
-
-void _cinnamon_link_to_bluetooth(void) {
-  bluetooth_applet_get_type();
-}
-#endif
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -266,7 +266,6 @@
 libcinnamon_la_LIBADD =		\
 	-lm			\
 	$(CINNAMON_LIBS)	\
-	$(BLUETOOTH_LIBS)	\
 	libst-1.0.la       	\
 	libtray.la		\
 	libgvc.la		\
